packages/react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -466,15 +482,26 @@ class Dropdown extends AutoControlledComponent<Extendable<DropdownProps>, Dropdo
             content,
             fluid: true,
             styles: styles.triggerButton,
-            ...getToggleButtonProps({
-              onFocus: this.handleTriggerButtonOrListFocus,
-              onBlur: this.handleTriggerButtonBlur,
-              onKeyDown: e => {
-                this.handleTriggerButtonKeyDown(e, rtl)
-              },
-              'aria-label': content,
-            }),
+            ...restToggleButtonProps,
           },
+          overrideProps: (predefinedProps: IconProps) => ({
```

Comment:
@silviuavram can you take a look on this changes? They should fix the issue: https://github.com/stardust-ui/react/issues/1329

Reply:
I think there is an issue created by me to refactor these handlers, probably this is the way to go. It was confusing for me at the time. We have user handler, stardust behavior, Downshift handler.

https://github.com/stardust-ui/react/issues/1254

Reply:
Yep, maybe they are duplicated. At least those should be resolved with these changes.